<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drowsypick_toolClass</class>
 <widget class="QWidget" name="Drowsypick_toolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drowsypick_tool</string>
  </property>
  <widget class="QPushButton" name="PLAY">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_right">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>220</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="CameraView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>651</width>
     <height>423</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>928</width>
     <height>423</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>32</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_left">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_left">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>561</y>
     <width>256</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_right">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>551</y>
     <width>256</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Drowsypick_tool.qrc"/>
 </resources>
 <connections/>
</ui>
